### PR TITLE
allow arbitrary config params with more efficient repr

### PR DESCRIPTION
**Pull request by conradludgate**

### CODE REVIEW

File: tokio-postgres/src/config.rs [hunk 1]
```diff
@@ -588,17 +579,6 @@ impl Config {
                 };
                 self.channel_binding(channel_binding);
             }
-            "replication" => {
```
Comment (khanova) [inline tokio-postgres/src/config.rs:591]: Why do we remove it from there?

**Review by khanova — APPROVED** [review]
LGTM.
Btw, do you think, we should merge origin into our fork at some point?

File: tokio-postgres/src/config.rs (hunk 1 above)
Comment (conradludgate) [reply tokio-postgres/src/config.rs:591]: The fallback branch will handle it

**Comment by conradludgate** [comment]
> Btw, do you think, we should merge origin into our fork at some point?

Yeah, although much easier said than done. 😭

**Pull request merged**